### ISSUE TITLE
filenode exception when chunkshape is None

### ISSUE THREAD

**Issue by DeadParrot** [PyTables/PyTables#397]
In PyTables 3.1.1 the `readline` method of `RawPyTablesIO` has this line:
```python
chunksize = self._node.chunkshape[0]
```
that triggers an exception when `chunkshape` is `None`. This fixes it:
```python
chunksize = self._node.chunkshape[0] if self._node.chunkshape else -1
```

**Comment by FrancescAlted** [comment]
Where exactly is this code?  I cannot spot it in PyTables code.

**Comment by DeadParrot** [comment]
It is [here](https://github.com/PyTables/PyTables/blob/develop/tables/nodes/filenode.py#L286).

**Comment by FrancescAlted** [comment]
Ah, yep, that was a bit buried :)  Fixed in 32e34e7.  Thanks.

**Comment by DeadParrot** [comment]
Great! Thanks.
